lib/rubocop/cop/style/method_call_parentheses.rb

```diff
@@ -50,6 +46,13 @@ def same_name_assignment?(node)
               return false if asgn_node.send_type?
             end
 
+            if asgn_node.masgn_type?
+              mlhs_node, _mrhs_node = *asgn_node
+              vars = mlhs_node.children.map(&:children).flatten
```

Comment:
I meant that using splatting is faster than using `.children`. So in the interest of performance you ca split this expression in two expressions.